pallets/liquidity-pools-gateway/src/weights.rs

```diff
@@ -122,4 +124,26 @@ impl WeightInfo for () {
 			.saturating_add(RocksDbWeight::get().reads(2))
 			.saturating_add(RocksDbWeight::get().writes(1))
 	}
+
+	fn start_pack_messages() -> Weight {
+		// TODO: BENCHMARK CORRECTLY
+		//
+		// NOTE: Reasonable weight taken from `PoolSystem::set_max_reserve`
+		//       This one has one read and one write for sure and possible one
+		//       read for `AdminOrigin`
+		Weight::from_parts(30_117_000, 5991)
+			.saturating_add(RocksDbWeight::get().reads(1))
+			.saturating_add(RocksDbWeight::get().writes(1))
+	}
```

Comment:
TechDebt: We should factorize the weights of this file and use some auxiliary methods instead (similar to what William has done in its PR about freezing)

Reply:
Yup, we have a todo to use real weights for most of the gateway stuff.